### PR TITLE
[FxImporter] Add FxImporter config in e2e-test

### PR DESCRIPTION
**Pull request by penguin-wwy**

### CODE REVIEW

**Comment by penguin-wwy** [comment]
@stellaraccident

File: projects/pt1/python/torch_mlir_e2e_test/configs/__init__.py [hunk 1]
```diff
@@ -11,3 +11,4 @@
 from .stablehlo_backend import StablehloBackendTestConfig
 from .tosa_backend import TosaBackendTestConfig
 from .torchdynamo import TorchDynamoTestConfig
+from .fx_importer import FxImporterTestConfig
```
Comment (stellaraccident) [inline projects/pt1/python/torch_mlir_e2e_test/configs/__init__.py:14]: Could we call this file fx_importer_backend.py?

Too many same named modules in the project just make it hard to navigate and know which one you're opening/referencing.

**Review by stellaraccident — APPROVED** [review]
Nice/thanks! I'm ok landing this whenever you are happy that you have a starting point. Then we can enable in CI in a follow-up patch.

File: projects/pt1/python/torch_mlir_e2e_test/configs/__init__.py (hunk 1 above)
Comment (penguin-wwy) [reply projects/pt1/python/torch_mlir_e2e_test/configs/__init__.py]: done

**Comment by penguin-wwy** [comment]
> Nice/thanks! I'm ok landing this whenever you are happy that you have a starting point. Then we can enable in CI in a follow-up patch.

I think we can merge this PR first, and then gradually improve it by figuring out the reasons for each failed case. I suggest opening another PR to disable the torchdynamo tests in CI first, as they are no longer valuable.

**Review by stellaraccident — APPROVED** [review]
I'm going to go ahead and merge this and as you say, we can iterate on it more as we manipulate the CI. Thank you for your help.

**Comment by sjain-stanford** [comment]
Thanks @penguin-wwy for landing this. Very happy to see we now have a way of testing e2e flows using the TorchDynamo frontend.

**Pull request merged**